docs/content/en/docs-dev/installation/install-control-plane/installing-controlplane-on-ECS.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Installing Control Plane on ECS"
+linkTitle: "Installing Control Plane on ECS"
+weight: 2
+description: >
+  This page describes how to install control plane on ECS.
+---
+
+Currently, we provide the example of deploying Control Plane to ECS using terraform. 
+
+Please refer to the blog post :)
+[PipeCD best practice 02 - control plane on ECS]({{< ref "/blog/control-plane-on-ecs.md" >}} "PipeCD best practice 02 - control plane on ECS").
```

Comment:
Checked whether to work the link and it is OK ✅ 
ref: https://gohugo.io/content-management/cross-references/#use-of-ref-and-relref